cnapy/gui_elements/strain_design_dialog.py

```diff
@@ -1313,21 +1348,164 @@ def compute(self):
             return
         bilvl_modules = [i for i,m in enumerate(self.modules) \
                             if m[MODULE_TYPE] in [OPTKNOCK,ROBUSTKNOCK,OPTCOUPLE]]
-        if len(bilvl_modules) > 1:
-            QMessageBox.information(self, "Conflicting Modules",
-                                    "Only one of the module types 'OptKnock', " +\
-                                    "'RobustKnock' and 'OptCouple' can be defined per " +\
-                                    "strain design setup.")
-            self.current_module = bilvl_modules[0]
-            self.module_edit()
-            return
         sd_setup = self.parse_dialog_inputs()
-        self.launch_computation_signal.emit(json.dumps(sd_setup))
+
+        if self.solver_buttons['OPTLANG'].isChecked():
+            if len(bilvl_modules) > 0:
+                QMessageBox.information(self, "Bilevel modules not supported",
+                                        "Mmodule types 'OptKnock', " +\
+                                        "'RobustKnock' and 'OptCouple' are not supported " +\
+                                        "by optlang_enumerator.")
```

Comment:
In these optlang-specific error messages, it would be nice to direct the user to a different solver, e.g. "Please use a different solver such as GLPK to use these unsupported features.".